Sevenpros/Teamwork
# #168373669 build out delete interface

#### What does this PR do?
 build out delete interface
#### Description of Task to be completed?
add delete icon to bottom of article
design a modal for deleting the article
add delete button inside the created modal
add cancel button inside the create modal
fix some bugs from the previous committed branch
#### How should this be manually tested?
when user click on delete icon below the article
a modal should pop up to confirm the action
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168373669
